contracts/minter-suite/Minters/MinterDAExpSettlement/MinterDAExpSettlementV0.sol

```diff
@@ -98,9 +98,9 @@ contract MinterDAExpSettlementV0 is
         // max uint64 ~= 1.8e19 sec ~= 570 billion years
         uint64 timestampStart;
         uint64 priceDecayHalfLifeSeconds;
-        uint256 startPrice;
+        uint128 startPrice;
         // base price is non-zero for all configured auctions on this minter
-        uint256 basePrice;
+        uint128 basePrice;
```

Comment:
We should consider a documentation comment that clarifies what the max price is in ETH terms.

Reply:
good call, added a comment for this!